cmake/FindCGAL.cmake

```diff
@@ -56,5 +56,5 @@ else()
     message(STATUS "CGAL: found config at '${CGAL_DIR}'.")
 endif()
 
-target_compile_definitions(CGAL::CGAL INTERFACE IFOPSH_WITH_CGAL)
+target_compile_definitions(CGAL INTERFACE IFOPSH_WITH_CGAL)
```

Comment:
Let's keep it `CGAL::CGAL`? Since it is the target name defined above and also the target name exported by `CGAL` package and also exported by conan.